code/go/0chain.net/smartcontract/storagesc/challenge.go

```diff
@@ -653,15 +681,22 @@ func (sc *StorageSmartContract) challengePassed(
 				DataRead:          dataRead,
 			})
 		if err != nil {
-			return "", common.NewError("verify_challenge",
-				"can't add to ongoing partition list "+err.Error())
+			logging.Logger.Info("verify_challenge can't add to ongoing partition list " + err.Error())
```

Comment:
We shall never miss the error checking and return if any, as it will hide the `node not found` error and then cause state hash mismatch error.